app/src/pages/docs/index.module.sass

```diff
@@ -22,17 +21,18 @@
   pre,
   p,
   ul
-    margin-bottom: 16px
+    margin-bottom: 38px
 
   h1,
   h2,
   h3,
   h4,
   h5
-    margin-top: 64px
+    margin-top: 62px
     margin-bottom: 16px
     font-weight: 600
     line-height: 1.25
+    cursor: pointer
```

Comment:
Please remove `cursor: pointer` from heading styles. Use `pointer` only for the anchor element.